### DISCUSSION TITLE
Replace the Area chart with stacked bar chart in the report

### DISCUSSION THREAD

**Discussion by milanschwartz** [maester365/maester#519] [Ideas]
I suggest we use a stacked bar chart vs an area chat in the html report to better represent the data.

Current area chart:
<img width="684" alt="Screenshot 2024-11-05 at 09 25 40" src="https://github.com/user-attachments/assets/4188916e-488e-4954-aa4d-8d6b523a4c7f">

Proposed stacked chart:
<img width="827" alt="Screenshot 2024-11-05 at 09 23 11" src="https://github.com/user-attachments/assets/41ee14b7-fecb-4e9d-a17a-01b5d22900bd">

**Comment by Oppedijk** [comment]
I agree, but also wonder if a stacked chart will show the correct information. 

The number of failed tests might be low, but are more crucial. So maybe we need a different UI to warn of failures, and use this stack for progress on completed items?

**Comment by merill** [comment]
Agree, this needs to be shown in a different way. The x axis being the category does not convey much information here.
